plugins/base/src/test/kotlin/transformers/SuppressTagFilterTest.kt

```diff
@@ -88,6 +89,26 @@ class SuppressTagFilterTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun `should filter setter`() {
+        testInline(
+            """
+            |/src/suppressed/Suppressed.kt
+            |var property: Int
+            |/** @suppress */
+            |private set
+            """.trimIndent(), configuration
+        ) {
+            preMergeDocumentablesTransformationStage = { modules ->
+                val prop = modules.flatMap { it.packages }.flatMap { it.properties }
+                    .firstOrNull { it.name == "property" } as? DProperty
```

Comment:
Seems like you can get rid of `as?` if you use `find` (which uses the same `firstOrNull`)
```suggestion
                    .find { it.name == "property" }
```